app/routes/verfahrensUebersicht.tsx

```diff
@@ -0,0 +1,15 @@
+import PageNavigation from "~/components/PageNavigation";
+
+export default function VerfahrensUebersicht() {
+  return (
+    <div className="container">
+      <PageNavigation />
+      <hr className="kern-divider" aria-hidden="true" />
+      {/* TODO: For future reference, add new components here  */}
+      <h1 className="kern-heading-large">Verfahren</h1>
+      <hr className="kern-divider" aria-hidden="true" />
+      <h1 className="kern-heading-large">Suche</h1>
+      <h1 className="kern-heading-large">Verfahren lsit</h1>
+    </div>
+  );
+}
```

Comment:
We could also make use of the already defined `ContentPage` layout (and enable it to render the layout here) or create a new one, if necessary?

```suggestion
import ContentPage from "~/components/ContentPage";
import PageNavigation from "~/components/PageNavigation";

export default function VerfahrensUebersicht() {
  return (
    <>
      <PageNavigation />
      <ContentPage title="Verfahren">Work in progress</ContentPage>
    </>
  );
}
```
<img width="913" height="466" alt="Bildschirmfoto 2025-08-27 um 09 27 05" src="https://github.com/user-attachments/assets/5f9e412b-ac5d-46cb-8212-1cfea83f6b60" />

Reply:
As discussed earlier: Let's forget about the page layout things for now.